articles/entra-conditional-access-integration.md

```diff
@@ -56,11 +56,9 @@ The following steps need to be configured on the Fleet teams you want to enable
 
 To enroll macOS devices to Entra for Conditional Access you will need to configure Fleet to automatically install the "Company Portal" macOS application.
 
-The Company Portal macOS application can be downloaded from https://go.microsoft.com/fwlink/?linkid=853070.
+To configure automatic installation on your macOS hosts, head to **Software > Add software > Fleet-maintained**. Select `Company Portal` and check the **Automatic install** option.
 
-To configure automatic installation on your macOS hosts, head to **Software > Add software > Custom package**. Upload the `CompanyPortal-Installer.pkg` and check the **Automatic install** option.
-
-!['Company Portal.app' automatic install](../website/assets/images/articles/company-portal-automatic-734x284@2x.png)
+!['Company Portal.app' automatic install](../website/assets/images/articles/company-portal-automatic-2070x962@2x.png)
```

Comment:
This link does not match the updated image's filename, but changing it to the current filename will cause the website's build-static-content script to throw an error because the new filename does not match the website's image naming conventions. 
 
Can you rename the updated image to `company-portal-automatic-900x450@2x.png` and update this link?

```suggestion
!['Company Portal.app' automatic install](../website/assets/images/articles/company-portal-automatic-900x450@2x.png)
```